src/index.html

```diff
@@ -145,30 +139,6 @@ <h1>Pantry</h1>
         </div>
       </article>
     </section> -->
```

Comment:
At this point I think it is better to not attempt the extension of the modal and just use a hidden popup. For the sake of time I think this is a better option.